ruff.toml

```diff
@@ -21,7 +21,7 @@ ignore = [
     "RUF012",
 ]
 
-exclude = ["demos/", "src/*"]
+exclude = ["demos/", "src/*", "tests/*"]
```

Comment:
Better don't add  tests dir to exclude, but need disable silly rules

Reply:
Agreed.